Installation/cmake/modules/FindSuiteSparse.cmake

```diff
@@ -0,0 +1,437 @@
+## CMake file to locate SuiteSparse and its useful composite projects
```

Comment:
What is more: if we ship it ourselves, there is a license issue. In that case the license of that file is "BSD 3-clauses", and we are required to reproduce the license and copyright notices.